### PR TITLE
HRIS-155 [BugTask] Apply user avatar in the DTR Management page and DTR Summary

### PR DESCRIPTION
**Pull request by NiloJr-sun**
## Issue Link

https://framgiaph.backlog.com/view/HRIS-155

## Definition of Done

- [x] Fixed DTR Management avatar in mobile view
- [x] Fixed DTR Summary avatar in both mobile and desktop view
- [x] Updated TimeEntriesSummaryDTO to include user Avatar
- [x] Updated LeaveDTO to include user Avatar
- [x] Updated TimeEntryDTO to include user Avatar

## Notes
-N/A

## Pre-condition
- in the root directory run ``` docker compose up --build ```

## Expected Output
- In the DTR management page, mobile view should now display the avatar
- In the DTR summary page, mobile view should now display the avatar

## Screenshots/Recordings

https://user-images.githubusercontent.com/109492180/225583497-bcb9b453-3d33-450f-b580-2d4890364e85.mp4

### CODE REVIEW

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✅

File: api/DTOs/TimeEntryDTO.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public TimeEntryDTO(TimeEntry timeEntry, Leave? leave)
             WorkedHours = timeEntry.WorkedHours?.Remove(timeEntry.WorkedHours.LastIndexOf(':'));
             TrackedHours = timeEntry.TrackedHours.ToString(@"hh\:mm");
             User = timeEntry.User;
+            Avatar = $"{domain}/media/{timeEntry.User.ProfileImage!.CollectionName}/{timeEntry.User.ProfileImage!.FileName}";
```
Comment (arden-dave) [inline api/DTOs/TimeEntryDTO.cs:21]:
```suggestion
           User = new UserDTO(timeEntry.User, domain);
```
If `User` and `domain` is available in DTO, you can instead use `UserDTO` so you won't need to create new `Avatar` attribute

**Review by arden-dave — APPROVED** [review]
Approve. 👍

File: client/src/components/molecules/ListOfLeaveTable/columns.tsx [hunk 2]
```diff
@@ -18,14 +17,7 @@ export const columns = [
...
-        <Avatar
-          src={
-            getUserProfileLink(Number(props?.row?.original?.userId)).data?.specificUserProfileDetail
-              ?.avatarLink as string
-          }
-          size="base"
-          rounded="full"
-        />
+        <Avatar src={`${props?.row?.original?.avatar}`} size="base" rounded="full" />
         <div className="flex flex-col items-start">
```
Comment (abduljalilpalala) [inline client/src/components/molecules/ListOfLeaveTable/columns.tsx:21]: Destruct the `props` object.

File: client/src/components/molecules/ListOfLeaveTable/MobileDisclose.tsx [hunk 3]
```diff
@@ -56,11 +56,7 @@ const MobileDisclose: FC<Props> = ({ table, isLoading, error }): JSX.Element =>
                         >
                           <div className="flex items-center justify-between">
                             <div className="flex items-center space-x-2">
-                              <Avatar
-                                src={`https://placeimg.com/640/480/abstract/${row.id}`}
-                                size="base"
-                                rounded="full"
-                              />
+                              <Avatar src={`${row.original.avatar}`} size="base" rounded="full" />
                               <div className="flex flex-col items-start">
```
Comment (abduljalilpalala) [inline client/src/components/molecules/ListOfLeaveTable/MobileDisclose.tsx:60]: Apply destruct on to the `row` object.

File: client/src/components/molecules/DTRSummaryTable/columns.tsx [hunk 4]
```diff
@@ -22,11 +22,7 @@ export const columns = [
     footer: (info) => info.column.id,
     cell: (props) => (
       <div className="flex items-center space-x-2">
-        <Avatar
-          src={`https://placeimg.com/640/480/abstract/${props.row.id}`}
-          size="base"
-          rounded="full"
-        />
+        <Avatar src={`${props.row.original.user.avatarLink}`} size="base" rounded="full" />
         <div className="flex flex-col items-start">
```
Comment (abduljalilpalala) [inline client/src/components/molecules/DTRSummaryTable/columns.tsx:26]: Apply destruct on to the `props` object.

File: client/src/components/molecules/DTRSummaryTable/MobileDisclose.tsx [hunk 5]
```diff
@@ -51,7 +51,7 @@ const MobileDisclose: FC<Props> = ({ table, isLoading, error }): JSX.Element =>
                             <div className="flex items-center space-x-2">
                               <div className="flex items-center space-x-2">
                                 <Avatar
-                                  src={`https://placeimg.com/640/480/abstract/${row.id}`}
+                                  src={`${row.original.user.avatarLink}`}
                                   size="base"
```
Comment (abduljalilpalala) [inline client/src/components/molecules/DTRSummaryTable/MobileDisclose.tsx:55]: Apply destruct on to the `row` object.

File: client/src/components/molecules/DTRManagementTable/columns.tsx [hunk 6]
```diff
@@ -24,14 +24,7 @@ export const columns = [
...
-        <Avatar
-          src={
-            getUserProfileLink(Number(props.row.original.user.id)).data?.specificUserProfileDetail
-              ?.avatarLink
-          }
-          size="base"
-          rounded="full"
-        />
+        <Avatar src={`${props.row.original.user.avatarLink}`} size="base" rounded="full" />
         <div className="flex flex-col items-start">
```
Comment (abduljalilpalala) [inline client/src/components/molecules/DTRManagementTable/columns.tsx:28]: Apply destruct on to the `props` object.

File: client/src/components/molecules/DTRManagementTable/MobileDisclose.tsx [hunk 7]
```diff
@@ -80,7 +80,7 @@ const MobileDisclose: FC<Props> = ({ table, isLoading, error }): JSX.Element =>
                               <div className="flex items-center space-x-2">
                                 <div className="flex items-center space-x-2">
                                   <Avatar
-                                    src={`https://placeimg.com/640/480/abstract/${row.id}`}
+                                    src={`${row.original.user.avatarLink}`}
                                     size="base"
```
Comment (abduljalilpalala) [inline client/src/components/molecules/DTRManagementTable/MobileDisclose.tsx:84]: Apply destruct on to the `row` object.

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✅

**Pull request merged**